editoast/src/views/stdcm/mod.rs

```diff
@@ -76,12 +76,14 @@ struct STDCMRequestPayload {
     standard_allowance: Option<AllowanceValue>,
 }
 
+const TWO_HOURS_IN_SECONDS: f64 = 12.0_f64 * 60.0_f64 * 60.0_f64;
```

Comment:
Ooops, copy/paste.

```suggestion
const TWO_HOURS_IN_SECONDS: f64 = 2.0_f64 * 60.0_f64 * 60.0_f64;
```

Reply:
Done in https://github.com/osrd-project/osrd/pull/5700/commits/7230a9cc465b62eabcdd580a041e4db027cbe871. Thanks